circle.yml

```diff
@@ -16,7 +16,7 @@ test:
     - wget --retry-connrefused --no-check-certificate -T 30 http://localhost:9000
     #run tests
     - >
-      if [[ $(git rev-parse --abbrev-ref HEAD) == *"feat"* || $(git rev-parse --abbrev-ref HEAD) == *"fix"* || $(git rev-parse --abbrev-ref HEAD) == *"chore"* ]] ; then
+      if [[ $CIRCLE_BRANCH != *"master"* || !$CIRCLE_TAG ]] ; then
           export FEAT_TEST=true; else unset FEAT_TEST
```

Comment:
i would do 2 things:
* instead of exporting an environment variable i would just run the command from here:
```
if [[ $CIRCLE_BRANCH != *"master"* || !$CIRCLE_TAG ]] ; then
  FEAT_TEST=true npm run test:e2e
elif
  npm run test:e2e
fi
```
Why?

Since all the commands are run in separate shell on circleci your `export`s will get lost, this would cause side effects you aren't prepared of. An example issue: https://discuss.circleci.com/t/circleci-cannot-read-env-variables-defined-inside-script/13131/2

* I would leverage the `BROWSER` environment variable. Even though if we use `TEST_PROVIDER=sauce`, we could still use the `BROWSER`.

@molnosikristof what do you think?

Reply:
done

Reply:
@molnosikristof what do you think of the second point?

Reply:
To be honest, I do not see the point of this change, can you explain why is it necessary?